pkg/tenant/tenant_test.go

```diff
@@ -112,6 +115,10 @@ func (suite *tenantTestSuite) SetupSuite() {
 	suite.srv, err = tenant.New(conf)
 	assert.NoError(err, "could not create the tenant api server from the test configuration")
 
+	suite.tlssrv = httptest.NewUnstartedServer(nil)
+	suite.srv.Server = *service.New(conf.BindAddr, service.WithMode(conf.Mode), service.WithTestServer(suite.tlssrv))
+	suite.srv.Server.Register(suite.srv)
```

Comment:
This is where I attempted to convert the server into a TLS server so that the cookies would be set correctly; I think I'm 90% of the way there, just missing one thing but I can't figure out what.

Reply:
Ah, I think I know - I'm not running `ServeTLS` … let me see if that fixes it.